cmd/kubelet/app/server.go

```diff
@@ -57,8 +60,6 @@ import (
 	"k8s.io/kubernetes/pkg/capabilities"
 	"k8s.io/kubernetes/pkg/client/chaosclient"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
```

Comment:
this is getting complicate. Some import with `client` prefix are from client-go, some are not.